control/config.py

```diff
@@ -166,9 +166,16 @@ def use_legacy_defaults(version):
     Parameters
     ----------
     version : string
-        version number of the defaults desired. Currently only supports `0.8.3`. 
+        version number of the defaults desired. ranges from '0.1' to '0.8.4'. 
     """
-    if version == '0.8.3': 
+    first_digit = int(version[0])
+    second_digit = int(version[2])
+    if second_digit < 8:
```

Comment:
Will break with >=0.10.0

Maybe use setuptool's [`packaging.version.parse`](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python)?

Reply:
I found I could make a quick change that addresses this concern - does what I came up with look ok? alternatively, are you sure a dependency on the module you proposed ok to have? (is it always included in python?)

Reply:
It is an edge case. setuptools should not really be needed after install, or if you get the package from something else. The other option mentioned in that SO question is undocumented but standard lib `distutils.version`. `python-control` already uses that within the test suite.

Although your quick solution works, too. I would be fine with it. (*Not to imply that I have any authority. I am only here to contribute unwarranted suggestions.*)